helix-view/src/editor.rs

```diff
@@ -1098,8 +1101,12 @@ impl Editor {
         let id = if let Some(id) = id {
             id
         } else {
-            let mut doc = Document::open(&path, None, Some(self.syn_loader.clone()))?;
```

Comment:
Note: it's possible to open the file and get the metadata from it and pass `None` to the function, stopping language config all together. I personally think it's not a good idea to do so.